docs/cli-tutorial.md

```diff
@@ -0,0 +1,738 @@
+```
+
+Note that in all these commands we are piping the results into the search. If you don't include the pipe then you'll
+get the filter output, which can be interested to inspect to see exactly what is sent to the server.
+
+#### Filter by asset
+
+You can limit your search to only data with a particular asset, for example search just for 8-band analytic assets:
+
+```console
+planet data filter --asset ortho_analytic_8b_sr | planet data search PSScene -
+```
+
+Or 8-band assets that also have a UDM.
+
+```console
+planet data filter --asset ortho_analytic_8b_sr --asset udm2 | planet data search PSScene -
+```
+
+You can find the list of available assets in each Item Type Page, like 
+[available assets](https://developers.planet.com/docs/data/psscene/#available-asset-types) for PSScene. You can see
+[a table of all Item Types](https://developers.planet.com/docs/data/psscene/#available-asset-types), which links to 
+the page for each with their list of asset types.
+
+Note that the asset filter doesn't perform any validation, so if your searches aren't returning anything check to make
+sure you got the asset right, and it's valid for the item-types you're searching.
+
+#### Permission Filter
+
+The 'permission filter' is set to true by default, since most people want to search only for data they have access to
+and are able to download. But if you'd like to just get search Planet's catalog and get a sense of what is out there
+you can set the permission filter to false:
+
+```console
+planet data filter --permission false --asset ortho_analytic_8b_sr | planet data search PSScene -
+```
+
+#### Sort
+
+You can also specify the sorting with your searches. The default sort is ordered by the most recent published
+images. But you can also sort by `acquired`, which is often more useful. You can sort in ascending or 
+descending order. The options are are:
+
+ * 'acquired asc'
+ * 'acquired desc'
+ * 'published asc'
+ * 'published desc'
+
+The lets you do things like get the id of the most recent skysat image taken (that you have download access to):
+
+```console
+planet data filter | planet data search SkySatCollect --sort 'acquired desc' --limit 1 - 
+```
+
+And you can also just get the ID, using `jq`
+
+```console
+planet data filter | planet data search SkySatCollect --sort 'acquired desc' --limit 1 - | jq -r .id
+```
+
+#### Bringing it all together
+
+The cool thing is you can combine the data and search commands, to make calls like ordering the most recent skysat
+image that was published:
+
+
+```console
+planet orders request --name "SkySat Latest" --item-type SkySatCollect --bundle analytic \
+--id `planet data filter | planet data search SkySatCollect --sort 'acquired desc' --limit 1 - | jq -r .id` \
+| planet orders create
+```
+
+Or get the 5 latest cloud free images in an area and create an order that clips to that area, using 
+[geometry.geojson](https://gist.github.com/cholmes/378d050a263ae433ddbbb91c3439994b) from above:
```

Comment:
would be cool to host this on the sdk repo

Reply:
just seems more permanent and maybe professional than your personal GH account. Like having an office number with a secretary that answers instead of a cellphone? IDK 😆

Reply:
I'm halfway done with this, but it'll be better when I merge to main, so I'm going to just do another PR once this one is merged to finish it off.